docs/web/sdk-framework-overview.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Framework Overview
```

Comment:
I'm not sure about referring to _framework_ explicitly. We've instead been referring to the bundle of custom stuff the dev creates as a _library_ or _custom library_.

We could also merge this in with the `SDK Overview` page as well if that simplifies things? Or call this `SDK Concepts` maybe?

Reply:
Yeah... I've tucked in the word framework in a number of places without explcitly describing it. It's a concept which developers don't really need to accomplish their goals, I can definitely remove the references to it.
I don't think "Concepts" works for the content, it's more descriptive of SDK structure and capabilities than theoretical. However, I think it could be merged into the SDK overview nicely...

Reply:
Merged into the SDK overview